library/std/src/sys/pal/unix/stack_overflow.rs

```diff
@@ -186,34 +204,48 @@ mod imp {
         libc::stack_t { ss_sp: stackp, ss_flags: 0, ss_size: sigstack_size }
     }
 
+    /// # Safety
+    /// Mutates the alternate signal stack
+    #[forbid(unsafe_op_in_unsafe_fn)]
     pub unsafe fn make_handler(main_thread: bool) -> Handler {
         if !NEED_ALTSTACK.load(Ordering::Relaxed) {
```

Comment:
```suggestion
        if !NEED_ALTSTACK.load(Ordering::Acquire) {
```

Reply:
Hmm, I thought operations on the same location on the same thread are still ordered even if Relaxed? I'm happy to diff this though, just making sure!

Reply:
...Oh, wait, this gets read by other threads lol nevermind.